Problem: 'make memcheck' shows memory leaks

This is the output of 'make memcheck': 

```
~/work/hintjens/malamute-core :master$ make memcheck
CC     src/src_mlm_selftest-mlm_selftest.o
CCLD   src/mlm_selftest
libtool --mode=execute valgrind --tool=memcheck \
        --leak-check=full --show-reachable=yes --error-exitcode=1 \
        --suppressions=./src/.valgrind.supp \
        ./src/mlm_selftest
==22332== Memcheck, a memory error detector
==22332== Copyright (C) 2002-2011, and GNU GPL'd, by Julian Seward et al.
==22332== Using Valgrind-3.7.0 and LibVEX; rerun with -h for copyright info
==22332== Command: /home/ph/work/hintjens/malamute-core/src/.libs/lt-mlm_selftest
==22332==
Running malamute selftests...
* zbits: OK
* mlm_stream_simple: OK
* mlm_msg: OK
* mlm_server: OK
* mlm_client: OK
Tests passed OK
==22332==
==22332== HEAP SUMMARY:
==22332==     in use at exit: 112 bytes in 3 blocks
==22332==   total heap usage: 2,007 allocs, 2,004 frees, 910,705 bytes allocated
==22332==
==22332== 32 bytes in 1 blocks are indirectly lost in loss record 1 of 3
==22332==    at 0x4C29DB4: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22332==    by 0x5437C4E: s_node_new (czmq_prelude.h:494)
==22332==    by 0x5438483: zlistx_new (zlistx.c:113)
==22332==    by 0x543C8C7: zmsg_new (zmsg.c:60)
==22332==    by 0x543CB71: zmsg_recv (zmsg.c:98)
==22332==    by 0x4E38A1A: mlm_msg_recv (mlm_msg.c:389)
==22332==    by 0x4E3EB07: s_client_handle_protocol (mlm_client_engine.inc:857)
==22332==    by 0x543A730: zloop_start (zloop.c:794)
==22332==    by 0x4E3F0AC: mlm_client (mlm_client_engine.inc:895)
==22332==    by 0x54247C2: s_thread_shim (zactor.c:67)
==22332==    by 0x5B19E99: start_thread (pthread_create.c:308)
==22332==    by 0x514631C: clone (clone.S:112)
==22332==
==22332== 56 bytes in 1 blocks are indirectly lost in loss record 2 of 3
==22332==    at 0x4C29DB4: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22332==    by 0x543846F: zlistx_new (czmq_prelude.h:494)
==22332==    by 0x543C8C7: zmsg_new (zmsg.c:60)
==22332==    by 0x543CB71: zmsg_recv (zmsg.c:98)
==22332==    by 0x4E38A1A: mlm_msg_recv (mlm_msg.c:389)
==22332==    by 0x4E3EB07: s_client_handle_protocol (mlm_client_engine.inc:857)
==22332==    by 0x543A730: zloop_start (zloop.c:794)
==22332==    by 0x4E3F0AC: mlm_client (mlm_client_engine.inc:895)
==22332==    by 0x54247C2: s_thread_shim (zactor.c:67)
==22332==    by 0x5B19E99: start_thread (pthread_create.c:308)
==22332==    by 0x514631C: clone (clone.S:112)
==22332==
==22332== 112 (24 direct, 88 indirect) bytes in 1 blocks are definitely lost in loss record 3 of 3
==22332==    at 0x4C29DB4: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==22332==    by 0x543C8AF: zmsg_new (czmq_prelude.h:494)
==22332==    by 0x543CB71: zmsg_recv (zmsg.c:98)
==22332==    by 0x4E38A1A: mlm_msg_recv (mlm_msg.c:389)
==22332==    by 0x4E3EB07: s_client_handle_protocol (mlm_client_engine.inc:857)
==22332==    by 0x543A730: zloop_start (zloop.c:794)
==22332==    by 0x4E3F0AC: mlm_client (mlm_client_engine.inc:895)
==22332==    by 0x54247C2: s_thread_shim (zactor.c:67)
==22332==    by 0x5B19E99: start_thread (pthread_create.c:308)
==22332==    by 0x514631C: clone (clone.S:112)
==22332==
==22332== LEAK SUMMARY:
==22332==    definitely lost: 24 bytes in 1 blocks
==22332==    indirectly lost: 88 bytes in 2 blocks
==22332==      possibly lost: 0 bytes in 0 blocks
==22332==    still reachable: 0 bytes in 0 blocks
==22332==         suppressed: 0 bytes in 0 blocks
==22332==
==22332== For counts of detected and suppressed errors, rerun with: -v
==22332== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 2 from 2)
make: *** [memcheck] Error 1
```

--- a/src/mlm_client_engine.inc
+++ b/src/mlm_client_engine.inc
@@ -968,6 +968,7 @@ mlm_client_destroy (mlm_client_t **self_p)
         zsock_destroy (&self->msgpipe);
         zstr_free (&self->reason);
         zstr_free (&self->command);
+        zmsg_destroy (&self->content);
         free (self);
         *self_p = NULL;
     }

--- a/src/mlm_selftest.c
+++ b/src/mlm_selftest.c
@@ -19,7 +19,6 @@
 */
 
 #include "mlm_classes.h"
-#include "labs/zbits.h"
 
 int
 main (int argc, char *argv [])

